integration_test.go

```diff
@@ -43,14 +43,14 @@ func testCartographerMap(t *testing.T, svc slam.Service) {
 
 	pointcloud, _ := pointcloud.ReadPCD(bytes.NewReader(pcd))
 	t.Logf("Pointcloud points: %v", pointcloud.Size())
-	test.That(t, pointcloud.Size(), test.ShouldBeGreaterThanOrEqualTo, 100)
+	test.That(t, pointcloud.Size(), test.ShouldBeGreaterThanOrEqualTo, 45)
 }
 
 // Checks the cartographer position within a defined tolerance.
 func testCartographerPosition(t *testing.T, svc slam.Service, expectedComponentRef string) {
-	expectedPos := r3.Vector{X: -0.004, Y: 0, Z: -0.004}
+	expectedPos := r3.Vector{X: -0.004, Y: -0.004, Z: 0}
 	tolerancePos := 0.04
-	expectedOri := &spatialmath.R4AA{Theta: 0, RX: 0, RY: 1, RZ: 0}
+	expectedOri := &spatialmath.R4AA{Theta: 0, RX: 0, RY: 0, RZ: -1}
```

Comment:
well i guess we found why the map is "flipped"

Reply:
why?

Reply:
cartographer is writing the orientation with Z down(at least that's how I am reading it). I don't think the previous transforms took that into account

Reply:
Would it be more accurate to say, we've now found a symptom of the map flip? Root cause is still TBD (although probably has to do with the -y we set in the map

Reply:
if the -y gets removed then the position has to be rotated accordingly

Reply:
but this is a convo for another PR